src/Arcus.Scripting.Tests.Unit/Arcus.Scripting.AppService.tests.ps1

```diff
@@ -0,0 +1,112 @@
+﻿Import-Module -Name $PSScriptRoot\..\Arcus.Scripting.AppService -ErrorAction Stop
+
+InModuleScope Arcus.Scripting.AppService {
+    Describe "Arcus Azure App Service unit tests" {
+        Context "Setting an Azure App Service application setting" {
+            It "Providing an App Service Name that does not exist should fail" {
+                # Arrange
+                $ResourceGroupName = "rg-infrastructure"
+                $AppServiceName = "unexisting-appservice"
+                $AppServiceSettingName = "somesetting"
+                $AppServiceSettingValue = "somevalue"
+
+                Mock Get-AzWebApp {
+                    return $null
+                } 
+
+                # Act
+                { 
+                   Set-AzAppServiceSetting -ResourceGroupName $ResourceGroupName -AppServiceName $AppServiceName -AppServiceSettingName $AppServiceSettingName -AppServiceSettingValue $AppServiceSettingValue
+                } | Should -Throw -ExpectedMessage "No app service with name '$AppServiceName' could be found in the resource group '$ResourceGroupName'"
+
+                # Assert
+                Assert-VerifiableMock
+            }
+            It "Creating a new setting succeeds" {
```

Comment:
Nit: maybe we can be more descriptive in our test names? Like you did in the first test?